.github/workflows/continuous-integration-workflow.yml

```diff
@@ -107,6 +107,11 @@ jobs:
       - name: Tests
         working-directory: builddir
         run: ctest --output-on-failure
+      - name: Upload benchmark results
+        uses: actions/upload-artifact@v3
+        with:
+          name: benchmark-results
```

Comment:
FIXME: [use a different name per job](https://github.com/actions/upload-artifact#uploading-to-the-same-artifact)